pyrobosim/pyrobosim/utils/polygon.py

```diff
@@ -227,4 +226,4 @@ def sample_from_polygon(polygon, max_tries=100):
             return sample_x, sample_y
 
     warnings.warn(f"Exceeded max polygon samples samples: {max_tries}")
-    return None, None
+    
```

Comment:
This is not the same as this case returns a tuple `(None, None)` rather than a single `None`. If you want to make this change, you will need to verify in the other places that call it, e.g.

```
x_sample, y_sample = sample_from_polygon( ... )
```

and make sure that they handle this appropriately.